cls/SourceControl/Git/Utils.cls

```diff
@@ -322,6 +347,69 @@ ClassMethod SwitchBranch(targetBranchName As %String) As %Status
     quit $$$OK
 }
 
+ClassMethod PreSync() As %String
+{
+    set uncommittedFilesWithAction = ##class(SourceControl.Git.WebUIDriver).UncommittedWithAction().%Get("user")
+    quit ..GenerateCommitMessageFromFiles(uncommittedFilesWithAction)
+}
+
+/// Commits all the files as needed by the Sync operation 
+ClassMethod SyncCommit(Msg As %String) As %Status
+{
```

Comment:
I think the Sync option should also go through each item modified by the current user and stage it as well.

Reply:
@isc-pbarton I added code to stage the files upon calling sync